lib/network_utils.ml

```diff
@@ -51,6 +51,7 @@ let bonding_dir = "/proc/net/bonding/"
 let uname = ref "/usr/bin/uname"
 let dracut = ref "/sbin/dracut"
 let modinfo = ref "/sbin/modinfo"
+let temp_vlan = "xentemp"
```

Comment:
That's not the best place. This module is for interfacing with system networking, and this doesn't belong in that category. I would just put it in network_config.ml.